CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+* and describes the pull request and review workflow in detail, including the
+  OWNERS files and automated workflow tool.
+
+## Leadership Roles
+
+The [kubeflow/manifests](https://github.com/kubeflow/manifests) repo is owned by
+the [Kubeflow Manifests Working Group](https://github.com/kubeflow/community/blob/master/wgs.yaml).
+
+Kubeflow's [governance model](https://github.com/kubeflow/community/blob/master/wgs/wg-governance.md)
+includes a plethora of different leadership roles.
+This section aims to provide a clear description of what these roles mean for
+this repo, as well as set expectations from people with these roles and requirements
+for people to be promoted in a role.
+
+A Working Group lead is considered someone that has either the role of
+**Subproject Owner**, **Tech Lead** or **Chair**. These roles were defined by trying
+to provide different responsibility levels for repo owners. For the Manifests WG
+we'd like to start by treating *approvers* in the root [OWNERS](https://github.com/kubeflow/manifests/blob/master/OWNERS),
+as Subproject Owners, Tech Leads and Chairs. This is done to ensure we have a
+simple enough model to start that people can understand and get used to. So for
+the Manifests WG we only have Manifests WG Leads, which are the root approvers.
+
+The following sections will aim to define the requirements for someone to become
+a reviewer and an approver in the root OWNERS file (Manifests WG Lead).
+
+### Manifests WG Lead Requirements
+
+The requirements for someone to be a Lead come from the processes and work required
+to be done in this repo. The main goal with having multiple Leads is to ensure
+that in case there's an absence of one of the Leads the rest will be able to ensure
+the established processes and the health of the repo will be preserved.
+
+With the above the main pillars of work and responsibilities that we've seen for
+this repo throughout the years are the following:
+1. Being involved with the release team, since the [release process](https://github.com/kubeflow/community/tree/master/releases) is tightly intertwined with the manifests repo
+2. Testing methodologies (GitHub Actions, E2E testing with AWS resources etc)
+3. Processes regarding the [contrib/addon](https://github.com/kubeflow/manifests/blob/master/contrib) components
+4. [Common manifests](https://github.com/kubeflow/manifests/tree/master/common)  maintained by Manifests WG (Istio, Knative, Cert Manager etc)
+5. Community and health of the project
+
+Root approvers, or Manifests WG Leads, are expected to have expertise and be able
+to drive all the above areas. Root reviewers on the other hand are expected to
+have knowledge in all the above and have as a goal to grow into the approvers
+role by helping with reviews throughout the project.
+
+As with the rest of the Kubeflow projects, and the governance model, in order
+for someone to become an approver they have to first to be a reviewer.
+
+#### Reviewer requirements
```

Comment:
@kimwnasptd Nice job! We might consider adding something like these...6. Will provide timely review of issues and PRs 7. Will attend and be an active participant in the Manifest Working Group meetings.

Reply:
Good point, I think that might be actually part of the "community and health of the project". Would it be helpful to be specific on what community and health means by saying

```
Investing in the community and health of the project by being active participants in open issues, pull requests, and community meetings
```

Reply:
Maybe also taking @juliusvonkohout's [comment](https://github.com/kubeflow/manifests/pull/2452#issuecomment-1529943836), it can be edited to

```
Investing in the community and health of the project by being active participants in issues, pull requests, community meetings, and making meaningful technical contributions

```

Reply:
That is also fine.

Reply:
That's a good point. Maybe another section in the doc is appropriate here to determine inactivity and how to demote for both approver and reviewer roles